packages/paste-core/components/table/src/types.ts

```diff
@@ -1,39 +1,49 @@
 import * as PropTypes from 'prop-types';
 import {ValueOf} from '@twilio-paste/types';
 import {WidthOptions, isWidthTokenProp} from '@twilio-paste/style-props';
-import {TableAlignmentObject, TableLayoutObject, TableVerticalAlignmentObject} from './constants';
+import {TableHTMLAttributes} from 'react';
+import {TableAlignmentObject, TableLayoutObject, TableVariantObject, TableVerticalAlignmentObject} from './constants';
 
 export type TableAlignmentOptions = ValueOf<typeof TableAlignmentObject>;
 export type TableLayoutOptions = ValueOf<typeof TableLayoutObject>;
+export type TableVariantOptions = ValueOf<typeof TableVariantObject>;
 export type TableVerticalAlignmentOptions = ValueOf<typeof TableVerticalAlignmentObject>;
 
+export interface TableContextProps {
+  striped: boolean;
+}
+
 export interface TableProps {
```

Comment:
Not extending from the table HTML element here?

Reply:
Thanks for catching that.